diff --git a/node_modules/vuepress-theme-vdoing/components/Buttons.vue b/node_modules/vuepress-theme-vdoing/components/Buttons.vue
index da2fbb4..2df929b 100644
--- a/node_modules/vuepress-theme-vdoing/components/Buttons.vue
+++ b/node_modules/vuepress-theme-vdoing/components/Buttons.vue
@@ -74,11 +74,11 @@ export default {
           icon: 'icon-yejianmoshi',
           KEY: 'dark'
         },
-        {
-          name: '阅读模式',
-          icon: 'icon-yuedu',
-          KEY: 'read'
-        }
+        // {
+        //   name: '阅读模式',
+        //   icon: 'icon-yuedu',
+        //   KEY: 'read'
+        // }
       ],
       _scrollTimer: null,
       _textareaEl: null,
diff --git a/node_modules/vuepress-theme-vdoing/components/Footer.vue b/node_modules/vuepress-theme-vdoing/components/Footer.vue
index 9935fc1..9521aca 100644
--- a/node_modules/vuepress-theme-vdoing/components/Footer.vue
+++ b/node_modules/vuepress-theme-vdoing/components/Footer.vue
@@ -15,14 +15,8 @@
     </div>
 
     <!--Vdoing主题遵循MIT协议，完全开源且免费。如果您对主题的修改并不大，希望您保留主题的链接。-->
-    Theme by
-    <a
-      href="https://github.com/xugaoyi/vuepress-theme-vdoing"
-      target="_blank"
-      title="本站主题"
-    >Vdoing</a>
     <template v-if="footer">
-      | Copyright © {{ footer.createYear }}-{{ new Date().getFullYear() }}
+      Copyright © {{ footer.createYear }}-{{ new Date().getFullYear() }}
       <span
         v-html="footer.copyrightInfo"
       ></span>
diff --git a/node_modules/vuepress-theme-vdoing/components/Home.vue b/node_modules/vuepress-theme-vdoing/components/Home.vue
index 3e2dcdd..6c57e63 100644
--- a/node_modules/vuepress-theme-vdoing/components/Home.vue
+++ b/node_modules/vuepress-theme-vdoing/components/Home.vue
@@ -356,7 +356,7 @@ export default {
         margin-top 3rem
         img
           max-width 100%
-          max-height 240px
+          max-height 300px
           display block
           margin 2rem auto 1.5rem
         h1
diff --git a/node_modules/vuepress-theme-vdoing/layouts/Layout.vue b/node_modules/vuepress-theme-vdoing/layouts/Layout.vue
index 7b3558a..6d9355b 100644
--- a/node_modules/vuepress-theme-vdoing/layouts/Layout.vue
+++ b/node_modules/vuepress-theme-vdoing/layouts/Layout.vue
@@ -311,6 +311,7 @@ export default {
       } else {
         this.themeMode = 'light'
       }
+      // this.themeMode = 'light'
     },
     toggleThemeMode (key) {
       if (key === 'auto') {
diff --git a/node_modules/vuepress-theme-vdoing/styles/palette.styl b/node_modules/vuepress-theme-vdoing/styles/palette.styl
index b0a0155..463e82d 100644
--- a/node_modules/vuepress-theme-vdoing/styles/palette.styl
+++ b/node_modules/vuepress-theme-vdoing/styles/palette.styl
@@ -26,7 +26,7 @@
 // 颜色
 
 $bannerTextColor = #fff // 首页banner区(博客标题)文本颜色
-$accentColor = #11A8CD
+$accentColor = #60ad81
 $activeColor = #ff5722
 $arrowBgColor = #ccc
 $badgeTipColor = #42b983
